netbox/dcim/tests/test_api.py

```diff
@@ -2040,6 +2040,11 @@ def setUpTestData(cls):
             'master': devices[1].pk,
         }
 
+        cls.update_data = {
+            'name': 'Virtual Chassis 1',
+            'master': None,
+        }
+
```

Comment:
`update_data` is already defined above, so this isn't needed. To test setting `master` to None, we can instead add `'master': None` to one or more of the object definitions under `bulk_update_data`.

Reply:
You'll have to excuse my ignorance when it comes to Django and how it does tests, I'm mostly familiar with Flask (plus am new to this codebase). Looking back now it should have been pretty obvious that this would just reassign `update_data` rather than adding an additional test that updates data...